dice.js

```diff
@@ -0,0 +1,57 @@
+
+let userChoice = prompt("how many times??");
+
+
+let randomNumber = function(){
+
+   let number = Math.random();
+   let num1;
+// if's :
+if (number < 0.166){
+return num1 = 1;
+}
+
+else if (number < 0.33){
+return num1 = 2;
+}
+else if (number < 0.496){
+return num1 = 3;
+}
+
+else if (number < 0.67){
+return num1 = 4;
+}
+else if (number < 0.826){
+return num1 = 5;
+}
+
+else{
+return num1 = 6;
+}
+
+
+
+}
+
```

Comment:
Great work!!, Also there is another way to create random number within a range, just one line: 
``` Math.round(Math.random() * (max - min) + min);```